app/models/user.rb

```diff
@@ -1,5 +1,14 @@
 class User < ActiveRecord::Base
   has_secure_password
 
+  require 'random_hash'
```

Comment:
I *think* if you move this outside the class definition we won't have that weird file loading problem we were having before. Can you give it a try?